docs/resources/Guild.md

```diff
@@ -736,24 +736,24 @@ Create a new [channel](#DOCS_RESOURCES_CHANNEL/channel-object) object for the gu
-| permission_overwrites\*\*     | array of partial [overwrite](#DOCS_RESOURCES_CHANNEL/overwrite-object) objects | the channel's permission overwrites                                                                                                                                             |
-| parent_id                     | snowflake                                                                      | id of the parent category for a channel                                                                                                                                         |
-| nsfw                          | boolean                                                                        | whether the channel is nsfw                                                                                                                                                     |
-| rtc_region                    | string                                                                         | channel [voice region](#DOCS_RESOURCES_VOICE/voice-region-object) id of the voice or stage channel, automatic when set to null                                                  |
-| video_quality_mode            | integer                                                                        | the camera [video quality mode](#DOCS_RESOURCES_CHANNEL/channel-object-video-quality-modes) of the voice channel                                                                |
-| default_auto_archive_duration | integer                                                                        | the default duration that the clients use (not the API) for newly created threads in the channel, in minutes, to automatically archive the thread after recent activity         |
-| default_reaction_emoji        | [default reaction](#DOCS_RESOURCES_CHANNEL/default-reaction-object) object     | emoji to show in the add reaction button on a thread in a `GUILD_FORUM` channel                                                                                                 |
-| available_tags                | array of [tag](#DOCS_RESOURCES_CHANNEL/forum-tag-object) objects               | set of tags that can be used in a `GUILD_FORUM` channel                                                                                                                         |
-| default_sort_order            | integer                                                                        | the [default sort order type](#DOCS_RESOURCES_CHANNEL/channel-object-sort-order-types) used to order posts in `GUILD_FORUM` channels                                            |
+| Field                         | Type                                                                           | Description                                                                                                                                                                     | Channel Type                     |
```

Comment:
```suggestion
| Field                         | Type                                                                           | Description                                                                                                                                                                     | Channel Types                     |
```

It should be plural like in [application command structure](https://discord.com/developers/docs/interactions/application-commands#application-command-object-application-command-structure)